index.js

```diff
@@ -322,7 +318,12 @@ module.exports = (command, args, options) => {
 	// eslint-disable-next-line promise/prefer-await-to-then
 	spawned.then = (onFulfilled, onRejected) => handlePromise().then(onFulfilled, onRejected);
 	spawned.catch = onRejected => handlePromise().catch(onRejected);
-	spawned.cancel = () => processDone.cancel();
+	spawned.cancel = function () {
+		if (!spawned.killed) {
```

Comment:
I did `!spawned.killed` instead of `spawn.kill()` because doing that would give another error(prcess was killed with SIGTERM) and not the one with message "Command was canceled" and property canceled equal to `true`(I think this is the required thing)

Reply:
This sounds good but I think you should still check the return value of `spawned.kill()`. For example if the child process exited successfully (not killed) `spawned.kill()` will return `false`. In that case it was not canceled, and `canceled` should not be `true`.